tests/test_utils.py

```diff
@@ -86,6 +86,7 @@ def test_unflatten(self, shape):
         assert unflattened.shape == reshaped.shape
         assert np.array_equal(unflattened, reshaped)
 
+    @pytest.mark.skip('New behavior has no unsupported types')
```

Comment:
This test should be deleted, probably.

Reply:
If the PR is removing this functionality, then yes.